src/tcgwars/logic/impl/gen4/RisingRivals.groovy

```diff
@@ -1797,9 +2370,12 @@ public enum RisingRivals implements LogicCardInfo {
           move "Errand-Running", {
             text "Search your deck for a Trainer card, show it to your opponent, and put it into your hand. Shuffle your deck afterward."
             energyCost C, C
-            attackRequirement {}
+            attackRequirement {
+              assert my.deck : "Your deck is emtpy"
+            }
             onAttack {
-              damage 0
+              my.deck.search("Search your deck for a trainer card",cardTypeFilter(ITEM)).showToOpponent("Errand-Running: Selected card").moveTo(my.hand)
```

Comment:
Maybe clarify that it should be the equivalent of an Item card here? This wording is going to be annoying either way though. I expect many invalid bug reports :laughing: